app/scenes/Talk/components/Pane/index.js

```diff
@@ -32,51 +34,114 @@ function Speaker({ data, onPress }) {
   return (
     <TouchableOpacity {...touchableProps}>
       <View style={styles.heroSpeaker}>
-        <Avatar source={data.avatar} />
+        <Avatar source={speaker.avatar} />
         <Text style={styles.heroSpeakerName}>
-          {data.name}
+          {speaker.name}
         </Text>
         <Text style={styles.heroSpeakerHint}>
           (tap for more)
         </Text>
       </View>
     </TouchableOpacity>
   );
-}
+};
 
 type Props = {
-  nextTalk?: ScheduleTalk,
-  nextTalkPreviewIsEngaged?: boolean,
+  nextTalk?: ScheduleTalk | null,
+  nextTalkPreviewIsEngaged: boolean,
   onHeroLayout?: (Object) => mixed,
   onPressNext?: (Object) => mixed,
-  onScroll?: (Object) => mixed,
+  onScroll?: ((Object) => mixed) | null,
   onScrollEndDrag?: () => mixed,
-  prevTalk?: ScheduleTalk,
-  prevTalkPreviewIsEngaged?: boolean,
-  showSpeakerModal: (SpeakerType | Array<SpeakerType>) => mixed,
+  prevTalk?: ScheduleTalk | null,
+  prevTalkPreviewIsEngaged: boolean,
+  showSpeakerModal: (SpeakerType) => mixed,
   visibleTalk: ScheduleTalk,
 };
 
+const TalkPreview = ({ talk, isEngaged }) => {
+  const speakers = talk.speakers.map(speaker => speaker.name).join(', ');
+
+  const subtitle = `${moment
+    .tz(talk.time.start, 'America/Los_Angeles')
+    .format(TIME_FORMAT)} - ${speakers}`;
+
+  return (
+    <Preview
+      isActive={isEngaged}
+      position="bottom"
+      subtitle={subtitle}
+      title={talk.title}
+    />
+  );
+};
+
+const TalkPreviewPrev = ({ talk, isEngaged }) => {
+  const preview = <TalkPreview talk={talk} isEngaged={isEngaged} />;
+
+  if (isAndroid) {
+    return null;
+  } else {
+    return (
+      <View ref="prevTalkPreview" style={{ opacity: 0 }}>
+        {preview}
+      </View>
+    );
+  }
+};
+
+const TalkPreviewNext = ({ talk, isEngaged, onPress }) => {
+  const preview = <TalkPreview talk={talk} isEngaged={isEngaged} />;
+
+  if (isAndroid) {
+    return (
+      <Animated.View style={{ opacity: this.state.animValue }}>
+        <TouchableHighlight
+          underlayColor={darken(theme.color.sceneBg, 10)}
+          onPress={onPress}
+        >
+          <View>
+            {preview}
+          </View>
+        </TouchableHighlight>
+      </Animated.View>
+    );
+  } else {
+    return (
+      <View ref="nextTalkPreview" style={{ opacity: 0 }}>
+        {preview}
+      </View>
+    );
+  }
+};
+
 export default class TalkPane extends Component {
   props: Props;
+
   state = {
     animValue: new Animated.Value(0),
   };
 
+  static defaultProps = {
+    nextTalkPreviewIsEngaged: false,
+    prevTalkPreviewIsEngaged: false,
+  };
```

Comment:
Using default props instead of making props optional simplifies a lot of types. As much as possible we should use these instead of `optional?: props`.